projects/storefrontlib/shared/components/carousel/focusable-carousel-item/focusable-carousel-item.directive.ts

```diff
@@ -51,7 +51,7 @@ export class FocusableCarouselItemDirective {
     const focusableTags = ['INPUT', 'SELECT', 'TEXTAREA', 'BUTTON', 'A'];
     const element = this.el.nativeElement;
     const hasTabIndex =
-      element.hasAttribute('tabindex') &&
+      element?.hasAttribute('tabindex') &&
```

Comment:
[QUESTION]
Is this change really needed? Or it's just a leftover from the initial debugging?